Test.groovy

```diff
@@ -3,7 +3,17 @@ pipeline {
   stages {
     stage('Build the docker images') {
       steps {
-        sh "docker-compose build"
+        sh "docker-compose -f docker-compose-tests.yaml build"
+      }
+    }
+    stage('Generate the database fixtures') {
+      steps {
+        script {
+          sh "rm -Rf fixtures && mkdir fixtures"
+          sh "chmod u+x ./fixtures-generator/entrypoint.sh"
+          // without that USER variable, it is not possible to delete the generated fixtures folder anymore
+          sh "USER=`id -u` docker-compose -f docker-compose-tests.yaml up fixtures-service"
+        }
       }
```

Comment:
The fixtures are generated separately as they are used in other tests than just the graphql-engine tests.

Reply:
The problem with the generated fixtures is that they are generated by default by the root user of the docker container, not by the jenkins agent. Consequently, the data produced in the fixtures folder belongs to root which prevents the jenkins agent from deleting them. To fix that, I let the agent run the `fixtures-service` by specifying 
```
USER=`id -u`
```
That particular syntax with back ticks was preferred over
```
USER=$(id -u)
```
which cannot be used here for some reasons (pipeline syntax makes it sometimes difficult).